src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM_SAIL = 20;
+  const LONG_TERM_SAIL = 50;
+  const CAR_RENT = 40;
+  const LONG_TERM_RENT = 7;
```

Comment:
The variable 'LONG_TERM_RENT' name is misleading. It seems to indicate a rent amount, but it holds the number of days for long term rent. Consider renaming it to 'LONG_TERM_RENT_DAYS'.